src/explorer/ConnectionTreeItem.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import { SiteClient } from 'vscode-azureappservice';
+import { IAzureNode, IAzureParentTreeItem, IAzureTreeItem } from 'vscode-azureextensionui';
+
+export class ConnectionTreeItem implements IAzureParentTreeItem {
```

Comment:
nit: This class/file should be named Connection**s**TreeItem

Reply:
What do you think about "Connection(s)Provider" instead of "ConnectionsTreeItem" and "CoonectionTreeItem" for nodes under this "ConnectionProvider"(similary to WebAppProvider and WebAppTreeItem classes).
![1100am](https://user-images.githubusercontent.com/15349723/45175652-c6b01380-b1c2-11e8-844a-fefd7d168bd9.PNG)
Or it's better to use "ConnectionsTreeItem" and "ConnectionTreeItem"(similary to "DeploymentSlotsTreeItem" and "DeploymentSlotTreeItem"?

Reply:
WebAppProvider is different. You'll notice it only implements `ChildProvider`, not `IAzureParentTreeItem`

I think ConnectionsTreeItem makes the most sense. For nodes under that, I would suggest "CosmosDBTreeItem" and "StorageTreeItem"